packages/babel-preset-env/src/targets-parser.js

```diff
@@ -194,35 +177,36 @@ const getTargets = (targets: Object = {}, options: Object = {}): Targets => {
   }
 
   // Parse browsers target via browserslist
-  const browsersquery = validateBrowsers(targets.browsers);
+  let browsersquery = validateBrowsers(targets.browsers);
 
   const hasTargets = Object.keys(targets).length > 0;
   const shouldParseBrowsers = !!targets.browsers;
   const shouldSearchForConfig =
     !options.ignoreBrowserslistConfig && !hasTargets;
 
-  if (shouldParseBrowsers || shouldSearchForConfig) {
-    // If no targets are passed, we need to overwrite browserslist's defaults
-    // so that we enable all transforms (acting like the now deprecated
-    // preset-latest).
-    //
-    // Note, if browserslist resolves the config (ex. package.json), then usage
-    // of `defaults` in queries will be different since we don't want to break
-    // the behavior of "no targets is the same as preset-latest".
-    if (!hasTargets) {
-      browserslist.defaults = objectToBrowserslist(targets);
+  if (!hasTargets) {
+    // If no targets are passed, we call loadConfig to resolve any external browserslist
+    // config.
+    if (shouldSearchForConfig) {
+      browsersquery = browserslist.loadConfig({
+        path: options.configPath,
+      });
     }
+    // If there is no external browserslist config and no targets are specified,
+    // we return an empty array so that "no targets and no browserslist is the same as
+    // preset-latest"
+    if (!browsersquery) {
+      browsersquery = [];
+    }
```

Comment:
I think that at this point it would make sense to default to `["defaults"]`?

Reply:
Well we can defaults to `defaults` here, need to tell people that if no targets are specified, it will always be `browserslists configuration ?? browserslist defaults`. And one can only use `forceAllTransforms: true` to apply all of them. I prefer `["defaults"]` here, let's see if people have different opinions.

Reply:
I thought that we were discussing about this during the last meeting :sweat_smile:

Reply:
Then I misunderstood the plans. It is now addressed in 8d5cef3.